ESM fails to load package

It seems a recent release has broken ESM usage (not CommonJS). Tested on version 4.1.0, this code no longer works with entities as the sole dependency:

```JavaScript
import entities from "entities"; // Crash!
console.log(entities);
```

<details>
<summary>Response</summary>

```
jiralite@MacBook-Pro-2021 test % node index.js
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/jiralite/Documents/GitHub/test/node_modules/entities/lib/esm/generated/encode-html' imported from /Users/jiralite/Documents/GitHub/test/node_modules/entities/lib/esm/encode-trie.js
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:418:11)
    at moduleResolve (node:internal/modules/esm/resolve:981:10)
    at defaultResolve (node:internal/modules/esm/resolve:1078:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```
</details>

Reverting to version 3.0.1 resolves this issue.

--- a/src/encode-trie.ts
+++ b/src/encode-trie.ts
@@ -1,4 +1,4 @@
-import htmlTrie from "./generated/encode-html";
+import htmlTrie from "./generated/encode-html.js";
 
 const enum Surrogate {
     Mask = 0b1111_1100_0000_0000,

